src/sentry/static/sentry/app/views/settings/components/forms/radioField.tsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import RadioGroup from 'app/views/settings/components/forms/controls/radioGroup';
-import InputField from 'app/views/settings/components/forms/inputField';
+import InputField, {onEvent} from 'app/views/settings/components/forms/inputField';
 
-class RadioField extends React.Component {
-  static propTypes = {
-    id: PropTypes.string,
-    value: PropTypes.string,
-    choices: PropTypes.arrayOf(PropTypes.array),
-    orientInline: PropTypes.bool,
-    disabled: PropTypes.bool,
-  };
+type RadioGroupProps = React.ComponentProps<typeof RadioGroup>;
+
+type Props = Omit<InputField['props'], 'type'> & {
+  choices: RadioGroupProps['choices'];
```

Comment:
@markstory Nit: why not use the `Pick` here on `RadioGroupProps`?

Reply:
I could do that, and union it with other props.